flow/flow.h

```diff
@@ -49,7 +49,16 @@
 
 using namespace std::rel_ops;
 
-#define TEST( condition ) if (!(condition)); else { static TraceEvent* __test = &(TraceEvent("CodeCoverage").detail("File", __FILE__).detail("Line",__LINE__).detail("Condition", #condition)); }
+#define TEST(condition)                                                                                                \
+	if (!(condition))                                                                                                  \
+		;                                                                                                              \
```

Comment:
If split into multiple lines, it may look visually more appealing to use braces instead of a dangling semicolon.

Reply:
Done